workspaces/mi/mi-extension/src/visualizer/activate.ts

```
@@ -343,6 +343,62 @@ export function activateVisualizer(context: vscode.ExtensionContext, firstProjec
+                            );
+                            switch (confirmUpdate) {
+                                case "Update Swagger":
+                                    rpcManager.updateSwaggerFromAPI({
+                                        apiName: apiName,
+                                        apiPath: apiPath,
+                                        existingSwagger: response.existingSwagger,
+                                        generatedSwagger: response.generatedSwagger
+                                    });
+                                    break;
+                                case "Update API":
+                                    const resources = getResources(st);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)